diff --git a/Makefile b/Makefile
index cc8ef82..327c439 100644
--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,7 @@ CXXFLAGS = -O2 -g
 INCLUDES = $(shell pkg-config --cflags libxml-2.0 rpm ossp-uuid)
 LIBS = $(shell pkg-config --libs libxml-2.0 rpm ossp-uuid) -lossp-uuid++ -ljsoncpp -lpthread
 TEST_INCLUDES = $(shell pkg-config --cflags cppunit libpcrecpp)
-TEST_LIBS = $(shell pkg-config --libs cppunit libpcrecpp)
+TEST_LIBS = $(shell pkg-config --libs cppunit libpcrecpp libxml-2.0 rpm)
 
 CXXWARNFLAGS = -Wall -Wno-long-long -Wextra -Weffc++ -pedantic -Wold-style-cast -Woverloaded-virtual -Wctor-dtor-privacy -Wreorder -Wabi -Wstrict-null-sentinel -Wc++0x-compat -Wformat=2 -Winit-self -Wswitch-default -Wsign-conversion -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wmissing-format-attribute -Wredundant-decls -Winline -Wsign-promo -Wunreachable-code
 
